README.md

```diff
@@ -1,11 +1,109 @@
 # compute_horde_sdk
 &nbsp;[![Continuous Integration](https://github.com/backend-developers-ltd/compute-horde-sdk/workflows/Continuous%20Integration/badge.svg)](https://github.com/backend-developers-ltd/compute-horde-sdk/actions?query=workflow%3A%22Continuous+Integration%22)&nbsp;[![License](https://img.shields.io/pypi/l/compute_horde_sdk.svg?label=License)](https://pypi.python.org/pypi/compute_horde_sdk)&nbsp;[![python versions](https://img.shields.io/pypi/pyversions/compute_horde_sdk.svg?label=python%20versions)](https://pypi.python.org/pypi/compute_horde_sdk)&nbsp;[![PyPI version](https://img.shields.io/pypi/v/compute_horde_sdk.svg?label=PyPI%20version)](https://pypi.python.org/pypi/compute_horde_sdk)
 
+## Installation
+
+```
+pip install compute-horde-sdk
+```
+
 ## Usage
 
+Register at https://facilitator.computehorde.io and generate an API token.
+
 > [!IMPORTANT]
 > This package uses [ApiVer](#versioning), make sure to import `compute_horde_sdk.v1`.
 
+Simple example:
+
+```python
+import asyncio
+import bittensor
+from compute_horde_sdk.v1 import ComputeHordeClient, ExecutorClass
+
+wallet = bittensor.wallet(name="...", hotkey="...")
+
+compute_horde_client = ComputeHordeClient(
+    hotkey=wallet.hotkey,
+    facilitator_token="MY_COMPUTE_HORDE_FACILITATOR_API_TOKEN",
+)
+
+async def main():
+    # Create a job to run on the Compute Horde.
+    job = await compute_horde_client.create_job(
+        executor_class=ExecutorClass.always_on__llm__a6000,
+        job_namespace="SN123.0",
+        docker_image="my-username/my-image:latest",
+    )
+    
+    await job.wait(timeout=10 * 60)
+    
+    print(job.status)  # Should be "Completed".
+
+asyncio.run(main())
+```
+
+Advanced example:
+
+```python
+import asyncio
+import bittensor
+from compute_horde_sdk.v1 import ComputeHordeClient, ExecutorClass, HuggingfaceInputVolume, HTTPInputVolume, HTTPOutputVolume
+
+wallet = bittensor.wallet(name="...", hotkey="...")
+
+compute_horde_client = ComputeHordeClient(
+    hotkey=wallet.hotkey,
+    facilitator_token="MY_COMPUTE_HORDE_FACILITATOR_API_TOKEN",
```

Comment:
i'm not sure what happened but this must not be required, signatures perform the role of authentication now